app/src/ui/main-process-proxy.ts

```diff
@@ -155,6 +155,9 @@ export const getCurrentWindowZoomFactor = invokeProxy(
   0
 )
 
+/** Tell the main process to that a modal dialog has opened */
+export const dialogOpened = sendProxy('dialog-opened', 0)
```

Comment:
Following the convention from other event-related stuff, I think the function name should be `sendDialogDidOpen` and the event `dialog-did-open`:
```suggestion
export const sendDialogDidOpen = sendProxy('dialog-did-open', 0)
```